BadApi.Database/dbo/Tables/Post.sql

```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE [dbo].[Post]
+(
+	[PostId] BIGINT NOT NULL IDENTITY, 
+    [AccountId] BIGINT NOT NULL, 
+    [Content] NVARCHAR(140) NOT NULL, 
+    [DateTimeCreatedUtc] DATETIME NOT NULL, 
+    CONSTRAINT [PK_Post] PRIMARY KEY ([PostId]) 
+)
```

Comment:
Should there be a FK constraint on AccountId to the Account table? Also an index on AccountId